Source/VCGeneration/Split.cs

```diff
@@ -127,7 +127,7 @@ void ObjectInvariant()
 
       // async interface
       private Checker checker;
-      private int splitNum;
+      private int splitIndex;
```

Comment:
What's wrong with `splitNum`?

Reply:
This is subjective, but to me `splitIndex` makes it more clear that this is the index that the split has in an ordered list of splits.